crates/nu-command/src/strings/str_/expand.rs

```diff
@@ -61,6 +66,19 @@ impl Command for SubCommand {
                 },)
             },
 
+            Example {
+                description: "If the piped data is path, you may want to use --path flag, or else manually replace the backslashes with double backslashes.",
+                example: "'C:\\{Users,Windows}' | str expand --flag",
+                result: Some(Value::List{
+                    vals: vec![
+                        Value::test_string("apple"),
+                        Value::test_string("banana"),
+                        Value::test_string("cherry")
```

Comment:
I'd be surprised if this expression 'C:\{Users,Windows}' returns apple, banana, cherry. 😆

Reply:
Thanks

Reply:
BTW, why the tests passed then?